custom-connectors/HR Cloud/apiDefinition.swagger.json

```diff
@@ -0,0 +1,829 @@
+                "properties": {
+                  "Id": {
+                    "type": "string",
+                    "description": "Division Id"
+                  },
+                  "xDivisionCode": {
+                    "type": "string",
+                    "description": "Division Code"
+                  },
+                  "xDivisionName": {
+                    "type": "string",
+                    "description": "Division Name"
+                  },
+                  "xRecordStatus": {
+                    "type": "string",
+                    "description": "Division Record Status"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Gets Division Data",
+        "description": "Gets Division Data",
+        "operationId": "GetDivision",
+        "parameters": [
+          {
+            "name": "filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "xDivisionName eq 'ENTER DIVISION NAME HERE'",
+            "x-ms-summary": "FilterDivisionName",
+            "description": "If you want to fetch a specific Division, enter the Division name in the place provided for that. If you want to fetch all the Division empty this field."
+          }
+        ]
+      }
+    },
+    "/v1/cloud/xEmploymentStatus": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "Id": {
+                    "type": "string",
+                    "description": "Employment Status Id"
+                  },
+                  "xRecordStatus": {
+                    "type": "string",
+                    "description": "Employment Status Record Status"
+                  },
+                  "xType": {
+                    "type": "string",
+                    "description": "Employment Status Type"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Gets Employment Status Data",
+        "description": "Gets Employment Status Data",
+        "operationId": "GetEmploymentStatus",
+        "parameters": [
+          {
+            "name": "filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "default": "xType eq 'ENTER EMPLOYMENT STATUS TYPE HERE'",
+            "x-ms-summary": "FilterEmploymentStatusType",
+            "description": "If you want to fetch a specific Employment Status, enter the Employment Status type in the place provided for that. If you want to fetch all the Employment Statuses empty this field."
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "undefined": {
```

Comment:
Please fix this from undefined to something meaningful. See an example here: https://github.com/microsoft/PowerPlatformConnectors/tree/dev/certified-connectors/AgilePoint%20NX